### ISSUE TITLE
Unable to use keyboard to navigate to Release Notes button

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#330046]
Testing #329291

1. Tab until focus is on the Update button.
2. A pop-up with buttons shows up.
3. :bug: I'm unable to tab or down arrow to the Release Notes button.

**Comment by sbatten** [comment]
When I use `Cmd+K, Cmd+I` while the update button is focused, the keybind is not recognized even though the keybinding is registered in the shortcuts editor.

**Comment by dmitrivMS** [comment]
@sbatten I am unable to repro... could you please clarify environment/repro steps?
![Image](https://github.com/user-attachments/assets/4c5b667c-1dd1-4d8f-8171-b8de449a9577)

**Comment by sbatten** [comment]
@dmitrivMS Exactly what you've done there, but on macOS.

https://github.com/user-attachments/assets/cdd34a94-7226-4c05-afa4-89b3c9ae4c3b

<img width="962" height="124" alt="Image" src="https://github.com/user-attachments/assets/546ee051-6234-4270-b2e2-6a638aecbb25" />
